grouper/repositories/permission_grant.py

```diff
@@ -178,14 +181,17 @@ def permission_grants_for_user(self, name):
 
         # Return the permission grants.
         group_permission_grants = (
-            self.session.query(Permission.name, PermissionMap.argument)
+            self.session.query(Permission.name, PermissionMap.argument, PermissionMap.granted_on)
             .filter(
                 Permission.id == PermissionMap.permission_id,
                 PermissionMap.group_id.in_(seen_group_ids),
             )
             .all()
         )
-        return [PermissionGrant(g.name, g.argument) for g in group_permission_grants]
+        return [
+            PermissionGrant(g.name, g.argument, g.granted_on, is_alias=False)
```

Comment:
Tracing through the code, I think that the graph and sql versions of `permission_grants_for_user` differ -- it looks like the graph version drills down to [here](https://github.com/dropbox/merou/blob/c1a349429ef3e53b3b1be641f4ae631d05e6749d/grouper/graph.py#L255) which does properly expand permission aliases, but this sql version never takes aliases into account.

Reply:
Yes, indeed, good catch. I noticed that too but forgot to add a comment. It's on my to-do list to fix in a later diff (the SQL version of `permission_grants_for_user` isn't used yet).